### PR TITLE
58 create the page for the user

### LINKED ISSUE #58: Create the page for the user
Just create the user's page. You do not need to connect any backend to it yet. 

users-page

### PR DESCRIPTION
**Pull request by shanqingwang**
# Resolves #58 

### What were your changes
Created basic user page functionality

### What specifically should be checked
Make sure user button works, page shows up and disappears as expected

### Next steps
Visual polish, collapsable lists, connect to backend per user

### CODE REVIEW

File: source/frontend/assets/components/UserPage.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        <button>Change Password</button>
+        <button>Change e-mail</button>
+      </div>
+    </div>
+
+    <div>
+      <button>EXPAND</button>
+      <t color=var(--font-color)>|</t>
+      <button>COLLAPSE</button>
+    </div>
```
Comment (joecai6) [inline source/frontend/assets/components/UserPage.js:119]: What is the expand and collapse button suppose to do? I tested it out on my side and they dont seem to work.

File: source/frontend/assets/components/UserPage.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    <div class="row">
+      <div class="column" style="align-self: left;">
+        <h2>User Name</h2>
+        <p>user-email</p>
+      </div>
+      <div class="column">
+        <button>Change Password</button>
+        <button>Change e-mail</button>
+      </div>
```
Comment (joecai6) [inline source/frontend/assets/components/UserPage.js:112]: @demyinn00  I dont think we have any routes to update the user info. should we remove this?

File: source/frontend/assets/components/UserPage.js (hunk 1 above)
Comment (shanqingwang) [reply source/frontend/assets/components/UserPage.js:119]: I believe they are ways to expand and collapse all 3 lists at once. I have not implemented the expanding and collapsing functionality of the lists yet.

File: source/frontend/assets/components/UserPage.js (hunk 2 above)
Comment (demyinn00) [reply source/frontend/assets/components/UserPage.js:112]: yes we should remove this

**Pull request merged**